### PR TITLE
Remove default info

### PR DESCRIPTION
**Pull request by awolant**
Removed default value info from docs strings as it is automatically propagated to docs.

### CODE REVIEW

File: dali/pipeline/operators/decoder/nvjpeg_decoder.cu [hunk 1]
```diff
@@ -31,12 +31,12 @@ Output of the decoder is on the GPU and uses `HWC` ordering.)code")
   .AddOptionalArg("device_memory_padding",
       R"code(Padding for nvJPEG's device memory allocations.
 This parameter helps to avoid reallocation in nvJPEG whenever a bigger image
-is encountered and internal buffer needs to be reallocated to decode it. Default is 16MB.)code",
+is encountered and internal buffer needs to be reallocated to decode it.)code",
```
Comment (JanuszL) [inline dali/pipeline/operators/decoder/nvjpeg_decoder.cu]: How about leaving some info that the units are in bytes?

File: dali/pipeline/operators/decoder/nvjpeg_decoder.cu [hunk 2]
```diff
@@ -31,12 +31,12 @@ Output of the decoder is on the GPU and uses `HWC` ordering.)code")
...
       16*1024*1024)
   .AddOptionalArg("host_memory_padding",
       R"code(Padding for nvJPEG's host memory allocations.
 This parameter helps to avoid reallocation in nvJPEG whenever a bigger image
-is encountered and internal buffer needs to be reallocated to decode it. Default is 16MB.)code",
+is encountered and internal buffer needs to be reallocated to decode it.)code",
```
Comment (JanuszL) [inline dali/pipeline/operators/decoder/nvjpeg_decoder.cu]: As above

File: dali/pipeline/operators/decoder/nvjpeg_decoder.cu (hunk 1 above)
Comment (awolant) [reply dali/pipeline/operators/decoder/nvjpeg_decoder.cu]: Done

File: dali/pipeline/operators/decoder/nvjpeg_decoder.cu (hunk 2 above)
Comment (awolant) [reply dali/pipeline/operators/decoder/nvjpeg_decoder.cu]: Done

**Comment by awolant** [comment]
Builds 616862

**Pull request merged**